bin/bioentitiestsvForGXAJSONIndex.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add a comment explaining what this is doing, step by step, so that when making changes later on is easy to interpret. Also, this seems to be dragging stuff from single cell that shouldn't be here. Please check the purpose of this and modify accordingly.

Reply:
This file is actually not used in bioentities, so removed it